client/layout/index.jsx

```diff
@@ -107,6 +128,7 @@ module.exports = React.createClass( {
 					<Welcome isVisible={ showWelcome } closeAction={ this.closeWelcome } additionalClassName="NuxWelcome">
 						<WelcomeMessage welcomeSite={ newestSite } />
 					</Welcome>
+					{ this.getAcceptInviteMessage() }
```

Comment:
I think it'd be nicer if all this was absorbed in a component/sub-component to not pollute `layout/index.jsx`.